src/concrete/ml/quantization/qat_quantizers.py

```diff
@@ -0,0 +1,25 @@
+"""Custom Quantiation Aware Training Brevitas quantizers."""
+from brevitas.quant.scaled_int import (
+    IntQuant,
+    MaxStatsScaling,
+    ParamFromRuntimePercentileScaling,
+    PerTensorPoTScaling8bit,
+    WeightQuantSolver,
+)
+from brevitas.quant.solver.act import ActQuantSolver
+
+
+# pylint: disable-next=too-many-ancestors
+class Int8ActPerTensorPoT(
```

Comment:
this needed a separate file because otherwise lazydocs failed when parsing these classes that inherit brevitas, so I needed to ignore only these classes in lazydocs .

Reply:
maybe add it as a comment somewhere then, it's not very obvious at first glance !